wordpress-parser/xml-to-svelte.js

```diff
@@ -22,34 +22,55 @@ async function parseAndWriteOutput({ inputGlob, outputDir, idToNameOutputFile, d
 	))
 
 	await Promise.all(fileContents.map(xmlString =>
-		parseXmlAndOutputSvelteComponents({ xmlString, outputDir, idToNameOutputFile, downloadImages, imageOutputDir })
+		parseXmlAndOutputSvelteComponents({ xmlString, outputDir, idToNameOutputFile, navigationOutputFile, downloadImages, imageOutputDir })
 	))
 }
 
-async function parseXmlAndOutputSvelteComponents({ xmlString, outputDir, idToNameOutputFile, downloadImages, imageOutputDir }) {
+async function parseXmlAndOutputSvelteComponents({ xmlString, outputDir, idToNameOutputFile, navigationOutputFile, downloadImages, imageOutputDir }) {
 	const doc = parseXml(xmlString)
 	const rss = findFirstChild(doc, `rss`)
 	const channel = findFirstChild(rss, `channel`)
 	const items = channel.children.filter(node => node.name === `item`)
 
+	const navigation = []
+	const categories = {}
+
 	const pageDetails = items.map(itemNode => {
 		const title = extractText(findFirstChild(itemNode, `title`))
 		const id = extractText(findFirstChild(itemNode, `wp:post_name`))
 		const content = extractText(findFirstChild(itemNode, `content:encoded`))
 
+		const category = findFirstChild(itemNode, `category`).attributes.nicename
+		const categoryTitle = extractText(findFirstChild(itemNode, `category`))
+
+		if(categories[category] === undefined) {
+			categories[category] = navigation.length
+			navigation.push({
+				id: category,
+				title: categoryTitle,
+				children: []
+			})
+		}
+
+		navigation[categories[category]].children.push(id)
+
 		return {
 			title,
 			id,
-			content,
+			content
```

Comment:
hah, I see you're not using my autoformat config ;-P

Reply:
haha, oops.  New computer, new editor config.